arrow-libs/optics/arrow-optics/src/main/kotlin/arrow/optics/std/tuple.kt

```diff
@@ -75,7 +75,7 @@ fun <A, B> Tuple2.Companion.second(): Lens<Tuple2<A, B>, B> = pSecond()
 @Deprecated(
   "Use the pairPTraversal function exposed in the Traversal' companion object",
```

Comment:
Should all the deprecated messages get updated with? :
```suggestion
  "Use the `pPair` function exposed in the `Traversal` companion object",
```